src/components/ReplaceServerDetails.js

```diff
@@ -188,6 +248,33 @@ class ReplaceServerDetails extends Component {
     }
   }
 
+  isCompute = () => {
```

Comment:
Please move this function into `ModelUtils`. It is clearly something that would be useful to have in several places in the code. It would be better to name it `isComputeNode` to make it clearer what we are talking about.